app/Http/Controllers/API/CSV/CSVImportController.php

```diff
@@ -29,31 +29,18 @@ public function store(Request $request, $id = null)
          * (primarily added because of OS9 line endings which do not work by default )
          */
         ini_set('auto_detect_line_endings', 1);
+        ini_set('memory_limit', '-1');
+        set_time_limit(720);
+
```

Comment:
Do we need these two if we are doing it in the job?

Reply:
oh yea we dont. nice catch